src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -414,6 +414,12 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
             }
         }
 
+        private string GetDecodedRouteParameter(RouteValueDictionary routeValues, string routeParameter)

```

Comment:
It looks like this method doesn't reference any instance variables or methods, so let's make it `static`. It's a good habit to get into because it makes the code a bit more efficient and it helps the reader quickly understand that a method doesn't depend on any external state (instance variables, etc.).